hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassCluster.java

```diff
@@ -164,6 +170,8 @@ private void readProperties() {
         keyspace = AlertProperties.getProperty(ALERTS_CASSANDRA_KEYSPACE, "hawkular_alerts");
         cassandraUseSSL = Boolean.parseBoolean(AlertProperties.getProperty(ALERTS_CASSANDRA_USESSL,
                 ALERTS_CASSANDRA_USESSL_ENV, "false"));
+        maxQueue = Integer.parseInt(AlertProperties.getProperty(ALERTS_CASSANDRA_MAX_QUEUE,
+                ALERTS_CASSANDRA_MAX_QUEUE_ENV, "9182"));
```

Comment:
I wouldn't suggest that we control this in an "efficient: or complicated way via singleton, etc..  I don't think it's worth the effort.   I think simple facades that can chunk via a configurable chunk size would probably be sufficient, despite not being a guarantee.  Filip is still seeing the busy pool issue despite @jsanda's introduction of the driver settings, so that is still a worry.  But, our use case is different than metrics, and I am OK if we don't pursue further at this time. Also, there may be improved driver handling in the future.